runtime/kusama/src/governance/old.rs

```diff
@@ -0,0 +1,189 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Old governance configurations for the Kusama runtime.
+//!
+//! Here purely so locked funds can be released before we purge the storage. It should be removed
+//! from the runtime once the migration was confirmed successful, probably in 1.1.
+//! See https://github.com/paritytech/polkadot/issues/6749
+
+use crate::*;
+use frame_support::{
+	parameter_types,
+	traits::{EitherOfDiverse, LockIdentifier},
+};
+use frame_system::EnsureNever;
+use static_assertions::const_assert;
+
+parameter_types! {
+	pub LaunchPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1, "KSM_LAUNCH_PERIOD");
+	pub VotingPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1 * MINUTES, "KSM_VOTING_PERIOD");
+	pub FastTrackVotingPeriod: BlockNumber = prod_or_fast!(3 * HOURS, 1 * MINUTES, "KSM_FAST_TRACK_VOTING_PERIOD");
+	pub const MinimumDeposit: Balance = 100 * CENTS;
+	pub EnactmentPeriod: BlockNumber = prod_or_fast!(8 * DAYS, 1, "KSM_ENACTMENT_PERIOD");
+	pub CooloffPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1 * MINUTES, "KSM_COOLOFF_PERIOD");
+	pub const InstantAllowed: bool = true;
+	pub const MaxVotes: u32 = 100;
+	pub const MaxProposals: u32 = 100;
+	pub MaxProposalWeight: Weight = Perbill::from_percent(50) * BlockWeights::get().max_block;
+}
+
+impl pallet_democracy::Config for Runtime {
```

Comment:
shouldn't the "old" democracy pallet be as close to unactivated as possible?

i.e. all the origins should be `EnsureNever`.

Reply:
Hmm I had not considered the hooks. You're right, ideally they would not run.

The issue wasn't with accessing storage but something a bit nastier, I'll revisit tomorrow.

Reply:
I'm not sure what you mean "implementing the configs for a made up `struct DepRuntime`", could you please elaborate?